src/app/static/src/tests/components/plots/bubble/sizeLegend.test.ts

```diff
@@ -7,11 +7,15 @@ import MapAdjustScale from "../../../../app/components/plots/MapAdjustScale.vue"
 import {ScaleType} from "../../../../app/store/plottingSelections/plottingSelections";
 import { mountWithTranslate, shallowMountWithTranslate } from "../../../testHelpers";
 
-vi.mock("@vue-leaflet/vue-leaflet", () => {
+vi.mock("@vue-leaflet/vue-leaflet", async () => {
+    const actual = await vi.importActual("@vue-leaflet/vue-leaflet")
```

Comment:
Why do you need this?

Reply:
so this lets you partially mock the import, unfortunately if i remove it then it errors saying that `LMap` is not defined in the Choropleth component even though we are only testing the SizeLegend, i have no idea but the cli hints at doing something like this so i just followed it